app/src/main/res/values/strings.xml

```diff
@@ -51,6 +51,8 @@
     <string name="quest_openingHours_chooseMonthsTitle">"Select months"</string>
     <string name="quest_openingHours_chooseWeekdaysTitle">"Select weekdays"</string>
     <string name="quest_openingHours_24_7_confirmation">"In other words, is it open any time, every day?"</string>
+    <string name="quest_openingHours_differs_by_day_of_week">Use the \"ADD OPENING HOURS\" button.\nSelect group of days with the same opening hours.\nFill opening hours.\nUse again \"ADD OPENING HOURS\"\nFill opening hours for the next group of days.\n\nRepeat if needed\n\nOnce completed use \"OK\" button.</string>
```

Comment:
Sounds a bit dry / unfriendly / complex.

Reply:
Yeah, I am not happy about it. I hope that someone will propose a better version.

Reply:
Hmm. Maybe

Reply:
How about changing "Select weekdays" to "Select weekdays with identical hours"?

Reply:
@goldfndr Great find, I completely missed it. I went with `Select group of weekdays with the same opening hours` - is it actually better, or is `Select weekdays with identical hours` preferable? I also reduced hint from dry and unfriendly to `You can use the "ADD OPENING HOURS" button multiple times, adding separate groups of days with the same opening hours every time.` what seems to be better.

Reply:
Your more verbose title is better. My only concern was whether the dialog will still fit on the screen, especially after translations, but if that's nothing to worry about…

In your hint, I'd change the portion after the comma to `adding a separate group of days with the same opening hours each time.` to make it clear that once or a few times might be adequate, to not overwhelm.

Is it possible to place an icon near (far right of) the title (e.g. a circled question mark) that would bring up the hint? That could reduce the need for verbosity.

Reply:
>  My only concern was whether the dialog will still fit on the screen, especially after translations